src/elements/content-sidebar/SidebarNavButton.scss

```diff
@@ -32,21 +32,19 @@
     }
 }
 
-.is-responsive-web {
-    @include breakpoint($medium-screen) {
-        .bcs-NavButton {
-            width: $sidebarTabResponsiveSize;
-            height: $sidebarTabResponsiveSize;
-            margin-right: $bdl-grid-unit * 3;
+@include breakpoint($medium-screen) {
+    .bcs .bcs-NavButton {
```

Comment:
I added the `bcs` class to match the non-breakpoint styles nesting and specificity